cmd/curio/tasks/tasks.go

```diff
@@ -186,8 +186,9 @@ func StartTasks(ctx context.Context, dependencies *deps.Deps) (*harmonytask.Task
 		if cfg.Subsystems.EnableSealSDRTrees {
 			treeDTask := seal.NewTreeDTask(sp, db, slr, cfg.Subsystems.SealSDRTreesMaxTasks)
 			treeRCTask := seal.NewTreeRCTask(sp, db, slr, cfg.Subsystems.SealSDRTreesMaxTasks)
+			treeSynth := seal.NewSyntheticProofTask(sp, db, slr)
```

Comment:
It's not a tree, more generating proofs
```suggestion
			genSynth := seal.NewSyntheticProofTask(sp, db, slr)
```